fri/src/verifier/mod.rs

```diff
@@ -378,3 +346,11 @@ fn get_query_values<E: FieldElement, const N: usize>(
 
     result
 }
+
+// Evaluates a polynomial with coefficients in an extension field at a point in the base field.
+pub fn eval_horner<E>(p: &[E], x: E) -> E
```

Comment:
The description of the method doesn't match it's actual behavior, as both `p` coefficients and the target point `x` are in the same field `E`. Plus when we're calling the horner evaluation in the verification, we're converting `x` from `E::BaseField` to `E`.

Reply:
Thank you! I updated it to be more consistent with my original way of thinking. Let me know if it works now.

Reply:
Thanks! I've added a comment, I think we can go down to a single generic parameter to make the function definition lighter.

Reply:
Thanks @Nashtare ! I looks cleaner indeed.

Reply:
Sorry for breaking the build, we need to update also when the function is called, to remove the extra `B` 😅

Reply:
Sure thing, I updated it now :-)